locator/abstract_replication_strategy.hh

```diff
@@ -44,6 +45,8 @@ using replication_strategy_config_options = std::map<sstring, sstring>;
 
 using replication_map = std::unordered_map<token, inet_address_vector_replica_set>;
 
+using endpoint_set = utils::basic_sequenced_set<inet_address, inet_address_vector_replica_set>;
```

Comment:
I'm not thrilled about sequenced_set, it was a type used to emulate some Java type, not something with deep meaning. But okay.